src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -37,15 +39,21 @@ public List<Resume> GetAll()
             List<Resume> resumes = new List<Resume>();
             using (IDbConnection db = new SqlConnection(this.connectionString))
             {
-                resumes = db.Query<Resume>("select * from resume").ToList();
+                resumes = db.Query<Resume>("select * from Resume").ToList();
```

Comment:
question, I thought we agreed to use IEnumerable at this level?
CC: @rodrigo-baldiviezo-jalafund @paola-quintanilla-jalafund

Reply:
@Raul-Gamarra-jalafund about this and just to double-check, are we going to use * instead of specifying the columns?